src/test/java/core/basesyntax/service/impl/WriterImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.Writer;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterImplTest {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String DESTINATION =
+            "src/test/resources/writerTestFile.csv";
+    private static final String EMPTY_TEST_FILE =
+            "src/test/resources/emptyTestFile.csv";
+    private static final String TEST_FILE =
+            "src/test/resources/testFile.csv";
+    private static final Writer writer = new WriterImpl();
+    private static String report;
+
+    @BeforeClass
+    public static void beforeClass() {
+        report = "type,fruit,quantity"
+                + LINE_SEPARATOR
+                + "b,pineapple,10"
+                + LINE_SEPARATOR
+                + "b,watermelon,87"
+                + LINE_SEPARATOR
+                + "b,lemon,30";
+    }
```

Comment:
if this report isn't changed, make it constant and initialize it instantly when declaring. i recomend you to declare dummy values at the test methods, as they could be different